tests/unit/CorrectCompanyName.spec.ts

```diff
@@ -12,25 +16,29 @@ function getLastEvent (wrapper: Wrapper<CorrectCompanyName>, name: string): any
   return null
 }
 
-describe.skip('CorrectCompanyName', () => {
+const defaultProps = {
+  companyName: 'Old Company Name',
+  formType: 'correct-new-nr',
```

Comment:
Can you use the enum here? (better typing)

Reply:
Yes, I can see